CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+## [Unreleased](https://github.com/jenkinsci/machine-learning-plugin/tree/HEAD)
+
+[Full Changelog](https://github.com/jenkinsci/machine-learning-plugin/compare/machine-learning-0.1-alpha-1...HEAD)
+
+**Fixed bugs:**
+
+- Agent execution in failing due to JVM Gateway  [\#25](https://github.com/jenkinsci/machine-learning-plugin/pull/25) ([Loghijiaha](https://github.com/Loghijiaha))
+
+**Merged pull requests:**
+
+- Added rest client for Jupyter notebook [\#26](https://github.com/jenkinsci/machine-learning-plugin/pull/26) ([Loghijiaha](https://github.com/Loghijiaha))
+- updated readme [\#24](https://github.com/jenkinsci/machine-learning-plugin/pull/24) ([Loghijiaha](https://github.com/Loghijiaha))
+- Need help to test [\#17](https://github.com/jenkinsci/machine-learning-plugin/pull/17) ([Loghijiaha](https://github.com/Loghijiaha))
```

Comment:
This doesn't need to match the PR title. You have to write text that is helpful to your plugin users. So that if they have a problem it may help them diagnose it, or so that they know what's changed.

Reply:
I have just used the commit messages. In the future, I will take care of commit messages so they can be useful for users. will change this right now. :+1: